src/frontends/tensorflow_common/src/helper_transforms/tensor_list_ops_resolver.cpp

```diff
@@ -64,6 +74,141 @@ ov::Rank find_element_rank(const ov::Input<ov::Node>& target_input) {
 
     return ov::Rank::dynamic();
 }
+
+bool get_scalar_constant_value(const ov::Output<ov::Node>& node, int64_t& output_value) {
```

Comment:
I see that we already have similar funciton in convert_ti_to_sequences.cpp 
could you move it to utils?

Reply:
Done